accounts/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from accounts.models import Account
+
+
+class SignUpForm(UserCreationForm):
```

Comment:
It's okay to add signup form for now, but we are creating this as an API so we'll need to use Serializer instead of form to accept payload from FE since registration will on FE and not on BE.

Simply add `CreateApiView` to the AccountSerializer and that will do it.
Additionally you can override `create` on that serializer to add extra checks on password and save it hash instead of plain text